Modules/_ssl.c

```diff
@@ -2834,6 +2841,50 @@ _ssl__SSLSocket_verify_client_post_handshake_impl(PySSLSocket *self)
 #endif
 }
 
+#if HAVE_EXPORT_KEYING_MATERIAL
+/*[clinic input]
+_ssl._SSLSocket.export_keying_material
+    label: Py_buffer(accept={buffer, str, NoneType})
+    key_len: int
+    context: Py_buffer(accept={buffer, str, NoneType})
```

Comment:
Why do you accept buffer, str, and None? Is this suppose to be optional text or bytes? The method should either accept text or bytes.